woocommerce/woocommerce-android
# Docs update

### Description

The PR updates the docs we have in the repository and fixes some issues with images

### Testing instructions

* Read the docs https://github.com/woocommerce/woocommerce-android/tree/docs-update/docs
* Notice if anything is outdated or more info need to be added
